authentication.ts

```diff
@@ -0,0 +1,65 @@
+import * as express from "express";
+import { createRemoteJWKSet, jwtVerify } from "jose";
+import { JwtError } from "@controllers/public-rest";
+
+// Create a JWKS client using Keycloak's JWKS endpoint
+const JWKS = createRemoteJWKSet(new URL(process.env.KEYCLOAK_CERTS!));
+
+export async function expressAuthentication(
+  request: express.Request,
+  securityName: string,
+  scopes?: string[]
+): Promise<any> {
+  console.log("authorization", request.header("authorization"));
```

Comment:
I'm afraid this will cause too much log spam on  the server.
Can we remove these logs and instead only log this if there was an unexpected error?

Or otherwise only log it in case of local setup using `if(NODE_ENV === "development")`